README.md

```diff
@@ -1,8 +1,12 @@
 # Opdrachtbeschrijving
 
 1. Clone deze opdracht naar jouw lokale machine
-2. Ontkoppel deze repo en koppel het project aan jouw eigen repo op GitHub
-3. Push jouw master
-4. Maak een featurebranch aan en maak dan het nep-huiswerk. Dat betekend: maak nog een paar html pagina's aan en voeg CSS toe.
-5. Commit jouw changes
-6. Push de featurebranch en maak een PR!
\ No newline at end of file
+2. Ontkoppel deze repo ($ git remote remove origin) en koppel het project aan jouw eigen repo op GitHub ($ git remote add origin https://github.com/JirosWorld/.....)
```

Comment:
Hier voeg ik de daadwerkelijke commando's toe, zodat ik die makkelijk meteen in de README zie.

Reply:
Wat super gaaf dat je dit hebt toegevoegd.